airbyte-integrations/connectors/source-postgres/src/main/java/io/airbyte/integrations/source/postgres/PostgresSource.java

```diff
@@ -265,6 +271,16 @@ protected static Properties getDebeziumProperties(JsonNode config) {
     return props;
   }
 
+  protected static String getTableWhitelist(ConfiguredAirbyteCatalog catalog) {
+    return catalog.getStreams().stream()
+        .map(ConfiguredAirbyteStream::getStream)
+        .map(AirbyteStream::getName)
+        // debezium needs commas escaped to split properly
```

Comment:
how does debezium handle quoted syntax for databases? anything special we need to do to handle it? or does it just convert to uft8 string?

Reply:
Added a test case. I'll also add an issue for properly handling quoting here long term.